tools/doc/hawqregister_help

```diff
@@ -0,0 +1,97 @@
+COMMAND NAME: hawq register
+
+Register parquet files generated by other system into the corrsponding table in hawq
+
+*****************************************************
+SYNOPSIS
+*****************************************************
+
+hawq extract [-h hostname] [-p port] [-U username] <databasename> <tablename> <hdfspath>
+
+hawq extract help 
+hawq extract -? 
+
+hawq extract --version
+
+*****************************************************
+DESCRIPTION
+*****************************************************
+
+Hawq register si a utility to register file(s) in hdfs into
+the table in hawq. It will move the file in the path(if path
+refers to a file) or files under the path(if path refers to a
+directory) into a new place corresponding to the table <tablename>,
+and then update the meatadata in the table.
```

Comment:
Hawq register si a utility to register file(s) in hdfs into the table in hawq. It will move the file in the path(if path refers to a file) or files under the path(if path refers to a directory) into a new place corresponding to the table <tablename>,
and then update the meatadata in the table.

=>

"hawq register" is a utility to register file(s) on HDFS into the table in HAWQ. It moves the file in the path (if path refers to a file) or files under the path (if path refers to a directory) into the table directory corresponding to the table <tablename>, and then updates the table meta data to include the files.